tests/TestCase/View/CellTest.php

```diff
@@ -207,7 +218,7 @@ public function testCellManualRenderError(): void
     public function testCellRenderThemed(): void
     {
         $this->View->setTheme('TestTheme');
-        $cell = $this->View->cell('Articles', ['msg' => 'hello world!']);
+        $cell = $this->View->cell('Articles');
```

Comment:
Do extra arguments fail now?

Reply:
Yes they do for named args, `['hello world!']` would run without errors.

Reply:
Looks good to me. We probably need a migration note incase people were sending extra args accidentally. PHP ignores extra positional arguments so that's probably the common case and this won't affect anyone.